### DISCUSSION TITLE
I'm having trouble authenticating my school ID.

### DISCUSSION THREAD

**Discussion by Moustafa345** [community/community#82100] [GitHub Education]
### Hi everyone,

Hi there! I need to take a picture of my school ID. The date is on the back rather than the front, and all of my information is on the front.
So what should I do to proof it?

**Comment by faridnec** [comment]
Hey no problem just try it with the front. I tried it recently - even with my card without date - yet they still accept it. Give it a go!

**Comment by iSiddharth20** [comment]
- Take Pictures of Both Sides.
- Create a PDF with both images in it.
- Upload this Document for Verification.
